Arrowgene.Ddon.GameServer/Characters/ItemManager.cs

```diff
@@ -234,7 +235,7 @@ private CDataItemUpdateResult ConsumeItem(DdonServer<GameClient> server, Charact
             return ntcData;
         }
 
-        public List<CDataItemUpdateResult> AddItem(DdonServer<GameClient> server, Character character, bool itemBag, uint itemId, uint num)
+        public List<CDataItemUpdateResult> AddItem(DdonServer<GameClient> server, Character character, bool itemBag, uint itemId, uint num, byte plusvalue = 0)
```

Comment:
Is there a better way we can reflect this plus value as input to this function? What about other types like color? equip points etc?

Reply:
I don't think it's too bad to have it as an argument. Even if color and points were added later it wont be such a long list of args. But yes we could be wrapping them all into a small struct and pass that instead.